mem0/memory/telemetry.py

```diff
@@ -1,10 +1,12 @@
 import platform
 import sys
+import os
 
 from posthog import Posthog
 
 from mem0.memory.setup import get_user_id, setup_config
 
+TELEMETRY = os.environ.get("TELEMETRY", True) 
```

Comment:
```suggestion
MEM0_TELEMETRY = os.environ.get("MEM0_TELEMETRY", True)
```

Reply:
Thanks for the suggestion @deshraj updated the code.